apps/daimo-bot/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@daimo/bot",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/api/index.ts",
+  "scripts": {
+    "build": "npx tsc",
+    "start": "node dist/api/index.js",
+    "dev": "nodemon src/api/index.ts",
+    "test": "tsc --noEmit && jest"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@daimo/api": "*",
+    "@daimo/common": "*",
```

Comment:
@dcposch @nalinbhardwaj 
FYI will need a fix, Vercel deployment fails with `'@daimo/api@*' is not in this registry`.
I looked into using git urls, but it fails because @daimo/api + @daimo/common aren't the top-level package in the [mono]repo

Reply:
Hm I think if you do `cd ../.. && npm run build` before your commands it should detect those packages. That's what we do for `daimo-web` which is also hosted on Vercel with similar dependencies:

<img width="923" alt="image" src="https://github.com/daimo-eth/daimo/assets/6984346/d2576098-5f63-4863-97f3-e8d41125e145">